src/OpenSage.Game/Graphics/ModelMesh.W3d.cs

```diff
@@ -238,7 +238,7 @@ private static ushort[] CreateIndices(W3dMesh w3dMesh)
                 switch (w3dShaderProperty.PropertyType)
                 {
                     case W3dShaderMaterialPropertyType.Texture:
-                        var texture = context.AssetStore.Textures.GetByName(w3dShaderProperty.StringValue);
+                        var texture = context.AssetStore.Textures.GetByName(w3dShaderProperty.StringValue) ?? context.StandardGraphicsResources.NullTexture;
```

Comment:
not all textures referenced by the maps are present in the big files

Reply:
The NullTexture doesn't seem to have any special color. Maybe a **SolidPinkTexture** would make sense, since that is the placeholder color used in the original titles aswell